src/Symfony/Bridge/PhpUnit/ExpectDeprecationTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit;
+
+use Symfony\Bridge\PhpUnit\Legacy\SymfonyTestsListenerTrait;
+
+trait ExpectDeprecationTrait
+{
+    private function expectDeprecation(string $message): void
```

Comment:
Added arg & return type because it's actually OK isn't it? We will only use this trait on >= 5.1. 3.4 won't use that trait as we won't add new deprecations there.

Reply:
the bridge is stated as compatible with PHP 5.5, so we need to stick to the declaration.

Reply:
what about making the method protected btw?